metapool/tests/test_prep.py

```diff
@@ -150,6 +226,27 @@ def test_preparations_for_run_missing_columns(self):
 
         self._check_run_191103_D32611_0365_G00DHB5YXX(obs)
 
+    def test_preparations_for_run_mf(self):
+        mf = pandas.read_csv(self.mf, delimiter='\t')
+
+        # self.mf has a header w/mixed case. This will test whether
+        # mapping-file headers are properly converted to all lower-case.
```

Comment:
self.mf doesn't exit, right?

Reply:
no it does - like self.ss, self.mf is a file path to an example mapping file and read_csv is using that path.

Reply:
right but the variable is not used in this test and it's referenced in the comment - which IMOO is confusing but it might also be that I don't fully understand the comment.

Reply:
Gotcha. I'll tweak the comment.